src/v2/components/UI/SplitPane/index.tsx

```diff
@@ -0,0 +1,160 @@
+  const [windowResize, setWindowResize] = useState(false)
+  const [isMobile] = useState(checkIfMobile())
+  const splitPane = createRef<any>()
+  const isFullScreen = layout === 'FULLSCREEN'
+  const minSize = isFullScreen ? 0 : panelSize || '28%'
+  const sizeAfterUpdate = windowResize ? panelSize : undefined
+
+  /**
+   * Add window resize handlers
+   */
+  useEffect(() => {
+    window.addEventListener('resize', updateSize)
+
+    return () => {
+      window.removeEventListener('resize', updateSize)
+    }
+  }, [])
+
+  /**
+   * Check to see if resize process is at rest
+   */
+  useEffect(() => {
+    if (windowResize) {
+      setWindowResize(false)
+    }
+  }, [isFullScreen, windowResize])
+
+  function updateSize() {
+    setWindowResize(true)
+  }
+
+  /**
+   * Update splitpane width percentages when dragging window
+   */
+  function handleDrag() {
+    const {
+      state: { draggedSize },
+      pane1: { clientWidth },
+    } = splitPane.current
+
+    if (draggedSize && clientWidth) {
+      const pct = draggedSize / window.innerWidth
+      const size = `${pct * 100}%`
+
+      setPanelSize(size)
+    }
+  }
+
+  /**
+   * Exit early if mobile
+   */
+  if (isMobile) {
+    return children
+  }
+
+  return (
+    <Container
+      context={context}
+      isFullScreen={isFullScreen}
+      panelSize={panelSize}
+    >
+      <ReactSplitPane
+        split="vertical"
+        primary={'second'}
+        allowResize={!isFullScreen}
+        ref={splitPane}
+        defaultSize={panelSize || minSize}
+        minSize={minSize}
+        size={isFullScreen ? '0%' : sizeAfterUpdate}
+        step={5}
+        onChange={handleDrag}
+        {...rest}
+      >
+        {children}
+      </ReactSplitPane>
+    </Container>
+  )
+}
+
+const Container = styled(Box)<{
+  context: LightboxContext
+  isFullScreen: boolean
+  panelSize: string | number
+}>`
+  position: absolute;
+  width: 100%;
+  height: ${props =>
+    props.context === 'PAGE'
+      ? `calc(100% - ${constants.topBarHeight})`
+      : '100%'};
+
+  /**
+    Styles for react-split-pane library
+   */
+  .Pane1 {
+    display: flex;
+  }
+
+  .Pane2 {
+    min-width: ${props => (props.isFullScreen ? 0 : 200)}px;
+  }
+
+  .Resizer {
+    cursor: ${props => (props.isFullScreen ? 'inherit' : 'col-resize')};
+    position: ${props => (props.isFullScreen ? 'absolute' : 'relative')};
+    padding: 10px;
+    opacity: 0;
+
+    &:hover {
+      opacity: 1;
+    }
+
+    &:after {
+      border-left: 1px solid ${props => props.theme.colors.gray.light};
+      content: '';
+      display: flex;
+      position: relative;
+      width: 1px;
+      height: 100%;
+    }
+  }
+
+  ${constants.media.mobile`
+    .SplitPane {
+      flex-direction: column !important;
+      overflow: scroll !important;
+
+      .Pane1,
+      .Pane2 {
+        width: 100% !important;
+      }
+      .Resizer {
+        display: none !important;
+      }
```

Comment:
Sorry for these `!importants`. I couldn't figure out how to "disable" `react-split-pane` any other way. When first rendering we can lean on the `isMobile` function helper, but if the user resizes the browser or rotates or whatever then we have to forcefully use CSS, via a media query, to return the display to its mobile form. Since `react-split-pane` uses inline styles to apply behavior, `!important` was the only way to override. 

A PR upstreram with a `disable` prop would fix this. Then, if true, it returns a pass-through component rather than initializing functionality. 

Happy for suggestions on how to fix.